src/core_plugins/kbn_vislib_vis_types/public/tile_map.js

```diff
@@ -1,9 +1,7 @@
-import _ from 'lodash';
 import supports from 'ui/utils/supports';
 import MapsVisTypeVislibVisTypeProvider from 'ui/vis_maps/maps_vis_type';
 import VisSchemasProvider from 'ui/vis/schemas';
 import AggResponseGeoJsonGeoJsonProvider from 'ui/agg_response/geo_json/geo_json';
-import FilterBarPushFilterProvider from 'ui/filter_bar/push_filter';
 import tileMapTemplate from 'plugins/kbn_vislib_vis_types/editors/tile_map.html';
 
 export default function TileMapVisType(Private, getAppState, courier, config) {
```

Comment:
it shouldn't zoom out when saving. The behavior should be:

- new visualizations are zoomed out all the way
- existing visualizations (saved ones, or the one you are currently working) should preserve the current zoom level. The idea is that when you open a saved-visualization, it reopens at the same location. Sharing a URL also makes it reopen at that saved location.

This is not the case?